views/content/dao.md

```diff
@@ -128,7 +128,6 @@ The way this particular democracy works is that it has an **Owner** which works
                members[id] = Member({member: targetMember, memberSince: now, name: memberName});
             } else {
                 id = memberId[targetMember];
-                Member m = members[id];
```

Comment:
@alexvandesande 
Thank you for your reply sande.
I tried you said, however still Ethereum wallet said that:


        Could not compile source code. 
        
        Unused local variable
            Member m = members[id];
            ^------^

And to me, it looks unnecessary.
Maybe my understanding of code above is not correct?
Or did I mistake something?  
  
<img width="1621" alt="2017-07-18 10 38 56" src="https://user-images.githubusercontent.com/2289647/28297086-e8bd95b4-6ba6-11e7-81f5-dd2cb6c65f36.png">
Ethereum Wallet v0.8.10